CMakeLists.txt

```diff
@@ -374,6 +381,18 @@ endif()
 
 set(CMAKE_MACOSX_RPATH TRUE)
 
+if( WITH_ASAN )
+  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -g -O1")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address -fno-omit-frame-pointer")
+endif()
+if( WITH_MSAN )
```

Comment:
could you please reflect in the logic that ```WITH_ASAN``` and  ```WITH_MSAN``` are mutually exclusive?

Reply:
Done